frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -0,0 +1,252 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import i18next from 'i18next';
+import { useTranslation } from 'react-i18next';
+import { compose } from 'redux';
+
+import { Status, useDeepCompareMemoize } from '@console/shared';
+import { sortable } from '@patternfly/react-table';
+import { Tooltip } from '@patternfly/react-core';
+import {
+  referenceForModel,
+  K8sResourceKind,
+  referenceFor,
+  ListKind,
+} from '@console/internal/module/k8s';
+import {
+  ListPage,
+  Table,
+  RowFunction,
+  RowFunctionArgs,
+  TableRow,
+  TableData,
+} from '@console/internal/components/factory';
+import {
+  resourcePathFromModel,
+  ResourceLink,
+  ResourceKebab,
+  Kebab,
+  humanizeBinaryBytes,
+} from '@console/internal/components/utils';
+import { usePrometheusQueries } from '@console/shared/src/components/dashboard/utilization-card/prometheus-hook';
+import { getInstantVectorStats } from '@console/internal/components/graphs/utils';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { StorageClassModel } from '@console/internal/models';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+
+import { menuActions } from './block-pool-menu-action';
+import { CephBlockPoolModel } from '../../models';
+import { StoragePoolKind, OcsStorageClassKind, CephClusterKind } from '../../types';
+import { CEPH_STORAGE_NAMESPACE, CEPH_EXTERNAL_CR_NAME } from '../../constants';
+import {
+  blockPoolColumnInfo,
+  isDefaultPool,
+  getScNamesUsingPool,
+  dateTimeFormatter,
+} from '../../utils/block-pool';
+import { PopoverHelper } from '../../utils/popover-helper';
+import { getPerPoolQueryMap } from '../../queries';
+import { COMPRESSION_ON } from '../../constants/storage-pool-const';
+import { cephClusterResource } from '../../resources';
+
+const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
+
+const getHeader = () => {
```

Comment:
```suggestion
const getHeader = (t) => {
```
Pass the return value of `useTranslation` here.

Reply:
ack